CSS/Default.sublime-keymap

```diff
@@ -71,4 +71,30 @@
 			{ "key": "following_text", "operator": "regex_contains", "operand": "^ \\}", "match_all": true }
 		]
 	},
+
+	// Auto-pair double quotes (also if followed by comma or semicolon)
+	// Example: key: |; -> key: "|";
+	{ "keys": ["\""], "command": "insert_snippet", "args": {"contents": "\"$0\""}, "context":
+		[
+			{ "key": "setting.auto_match_enabled", "operator": "equal", "operand": true },
+			{ "key": "selection_empty", "operator": "equal", "operand": true, "match_all": true },
+			{ "key": "selector", "operator": "equal", "operand": "source.css - string", "match_all": true },
+			{ "key": "following_text", "operator": "regex_contains", "operand": "^(?:\t| |\\)|]|\\}|>|,|;|$)", "match_all": true },
+			{ "key": "preceding_text", "operator": "not_regex_contains", "operand": "[\"a-zA-Z0-9_]$", "match_all": true },
+			{ "key": "eol_selector", "operator": "not_equal", "operand": "string.quoted.double - punctuation.definition.string.end", "match_all": true }
+		]
+	},
+
+	// Auto-pair double quotes (also if followed by comma or semicolon)
```

Comment:
"single quotes" here? :wink:

Reply:
I fixed this in 52546aaefbd5d12aa2df133f5691d414206bdc89

Reply:
Ouff. Haven't noticed that comment.